src/app.py

```diff
@@ -4212,6 +4213,155 @@ def multiple_components():
     return jsonify(normalized_complete_entity_list)
 
 
+def bulk_update_entities(
+    entity_updates: dict,
+    token: str,
+    entity_api_url: str,
+    total_tries: int = 3,
+    throttle: float = 5,
+    after_each_callback: Optional[Callable[[int], None]] = None,
+) -> dict:
+    """Bulk update the entities in the entity-api.
```

Comment:
Can we move the function comments above the function definition? Just to be consistent with others.

Reply:
This is not consistent with current Python conventions https://peps.python.org/pep-0257/ nor is it consistent with the way the the SenNet guys are coding, but I will move it since you have asked.

Reply:
@ChuckKollar you are right, we are not following the Python conventions nor SenNet stylings. It's just for the consistency sake and keeping the code organized inside HuBMAP entity-api.